### DISCUSSION TITLE
Incredible work! Thank you.

### DISCUSSION THREAD

**Discussion by jimi75** [zeux/meshoptimizer#288] [General]
Hi guys, I just want to say thank you. Incredible work with this library. Any way to donate or contribute other than PR's? This kind of low-level coding is far away from my knowledge

**Comment by zeux** [comment]
Thanks! I'm glad you like it. I don't have a donation stream associated with the repository - I went back and forth on this, but decided to not set it up in the end to keep things simple. PRs are of course always welcome, including ones that improve documentation; short of that, just tell your friends that this library is great, I take pleasure from seeing others use it and of course starring the repository :)
